themes-default/slim/views/manage_massEdit.mako

```diff
@@ -15,14 +15,49 @@
     allowed_qualities, preferred_qualities = common.Quality.split_quality(initial_quality)
 %>
 <script type="text/javascript" src="js/quality-chooser.js?${sbPID}"></script>
-<script type="text/javascript" src="js/mass-edit.js?${sbPID}"></script>
 <script>
 let app;
 const startVue = () => {
     app = new Vue({
```

Comment:
Warning while trying to load Vue.
```
vender.min.js?10032:2 jQuery.Deferred exception: Cannot set property '_renderItem' of undefined TypeError: Cannot set property '_renderItem' of undefined
    at ma.fn.init.$.fn.fileBrowser (http://localhost:8081/js/browser.js?10032:169:52)
    at Vue$3.mounted (http://localhost:8081/manage/massEdit:962:28)
    at callHook (http://localhost:8081/js/lib/vue.js:1802:19)
    at Vue$3.Vue._mount (http://localhost:8081/js/lib/vue.js:1676:7)
    at Vue$3.$mount (http://localhost:8081/js/lib/vue.js:5506:15)
    at Vue$3.$mount (http://localhost:8081/js/lib/vue.js:7381:16)
    at initRender (http://localhost:8081/js/lib/vue.js:2173:8)
    at Vue$3.Vue._init (http://localhost:8081/js/lib/vue.js:2515:5)
    at new Vue$3 (http://localhost:8081/js/lib/vue.js:2557:8)
    at startVue (http://localhost:8081/manage/massEdit:925:11) undefined
```
![image](https://user-images.githubusercontent.com/1331394/37874263-2d4c4898-302c-11e8-8aff-5b286a866184.png)

Reply:
This is the massEdit page.

Reply:
@p0psicles was this sorted?

Reply:
None of the buttons in Mass Update seem to work for me (`/manage`).